src/views/ManageList.jsx

```diff
@@ -1,7 +1,118 @@
-export function ManageList() {
+import { useState } from 'react';
+import { addItem } from '../api/firebase';
+import toast, { Toaster } from 'react-hot-toast';
+
+export function ManageList({ listPath }) {
+	console.log('listPath:', listPath);
+	const [itemName, setItemName] = useState('');
+	const [itemNextPurchaseTimeline, setItemNextPurchaseTimeline] = useState('');
```

Comment:
It would be beneficial to reset the state values after a successful form submission, so the form fields are cleared and ready for new input.